samples/mcuboot/README.rst

```diff
@@ -0,0 +1,40 @@
+.. _mcuboot_app:
```

Comment:
Looks like this is a new sample area, so you'll also need to edit ``samples/index.rst`` to add a line for ``mcuboot/*`` to the samples index page.

Reply:
Done

Reply:
@dbkinder : I believe that all feedback has been addressed.